android/WALT/app/src/main/java/org/chromium/latency/walt/MainActivity.java

```diff
@@ -339,6 +339,7 @@ private void attemptSaveAndShareLog() {
 
     @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
the superclass contains logic to route the callback to the fragment (specifically AudioFragment)